### PR TITLE
Add create kamino strategy

### PR DESCRIPTION
**Pull request by peroxy**
Current limitations:
- strategy can only be created by the owner (admin) of the global config, we will need to allow non-admins to bypass this check
- after the strategy is created, only the owner (admin) can update the treasury fee vault with token A/B, we need to allow non-admins to be able to do (and require) this as well - as we discussed with @silviutroscot 

There is a bunch of things that need to be discussed as well about what operations to allow non-admins to be able to do and manage their strategy: https://github.com/hubbleprotocol/yvaults/issues/192

### CODE REVIEW

File: packages/kamino-sdk/src/whirpools-client/programId.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import { PublicKey } from "@solana/web3.js"
+import { PublicKey } from '@solana/web3.js';
 
 // Program ID passed with the cli --program-id flag when running the code generator. Do not edit, it will get overwritten.
```
Comment (DemidovVladimir) [inline packages/kamino-sdk/src/whirpools-client/programId.ts:3]: Should we enable ability to pass some arguments by using .env?

File: packages/kamino-sdk/tests/kamino.test.ts [hunk 2]
```diff
@@ -194,4 +211,51 @@ describe('Kamino SDK Tests', () => {
   //   });
   //   console.log(txHash);
   // });
+
+  // test('should create a new strategy', async () => {
```
Comment (DemidovVladimir) [inline packages/kamino-sdk/tests/kamino.test.ts:215]: Is that intended?

File: packages/kamino-sdk/src/whirpools-client/programId.ts (hunk 1 above)
Comment (peroxy) [reply packages/kamino-sdk/src/whirpools-client/programId.ts:3]: I don't think it's neccessary. This is an NPM package (library), so the .env file and environment variables should be specified by the main application that is using this SDK. This way they can use whatever they want and use this new method to override the mainnet program ID, if needed.

File: packages/kamino-sdk/src/whirpools-client/programId.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
-export const PROGRAM_ID_CLI = new PublicKey(
-  "whirLbMiicVdio4qvUfM5KAg6Ct8VwpYzGff3uctyCc"
-)
+export const PROGRAM_ID_CLI = new PublicKey('whirLbMiicVdio4qvUfM5KAg6Ct8VwpYzGff3uctyCc');
```
Comment (y2kappa) [inline packages/kamino-sdk/src/whirpools-client/programId.ts:4]: call it whirlpool program_id

**Review by y2kappa — APPROVED** [review]

File: packages/kamino-sdk/src/whirpools-client/programId.ts (hunk 1 above)
Comment (DemidovVladimir) [reply packages/kamino-sdk/src/whirpools-client/programId.ts:3]: make sense

File: packages/kamino-sdk/src/whirpools-client/programId.ts (hunk 3 above)
Comment (peroxy) [reply packages/kamino-sdk/src/whirpools-client/programId.ts:4]: done

**Pull request merged**